### PR TITLE
Add AdminPanelModeratorsPage

### PR DESCRIPTION
**Pull request by LoveMyDaughter**

### CODE REVIEW

File: TAF_EasyRest/TestFramework/PageComponents/AdminPanelModeratorsPageComponent.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿namespace TestFramework.Pages
+{
+    public class AdminPanelModeratorsPageComponent : BasePage
+    {
+        public AdminPanelModeratorsPageComponent(IWebDriver driver) : base(driver) { }
+
+        #region Elements
```
Comment (sofiaso) [inline TAF_EasyRest/TestFramework/PageComponents/AdminPanelModeratorsPageComponent.cs:7]: Great! We should implement region in our progect

File: TAF_EasyRest/TestFramework/Pages/AdminPanelModeratorsPage.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+        public AdminPanelModeratorsPage(IWebDriver driver) : base(driver)
+        {
+            adminPanelModeratorsPageComponent = new AdminPanelModeratorsPageComponent(driver);
+        }
+
+
+
+
+        public AdminPanelModeratorsPage GoToUrl(string url)
```
Comment (sofiaso) [inline TAF_EasyRest/TestFramework/Pages/AdminPanelModeratorsPage.cs:26]: maybe, we should move it to basePage class instead

Comment (LoveMyDaughter) [reply TAF_EasyRest/TestFramework/Pages/AdminPanelModeratorsPage.cs:26]: Yes. So we'll have only one GoToUrl method in BasePage, right?

Comment (LoveMyDaughter) [reply TAF_EasyRest/TestFramework/Pages/AdminPanelModeratorsPage.cs:26]: And we shouldn't use GoToUrl in any other class?

Comment (sofiaso) [reply TAF_EasyRest/TestFramework/Pages/AdminPanelModeratorsPage.cs:26]: i think so

**Review by sofiaso — APPROVED** [review]

**Pull request merged**